client/lib/olark/index.js

```diff
@@ -60,9 +58,6 @@ const olark = {
 			this.getOlarkConfiguration()
 				.then( ( configuration ) => this.configureOlark( configuration ) )
 				.catch( ( error ) => this.handleError( error ) );
-		} else {
-			this.once( 'eligible', this.configureOlark.bind( this ) );
```

Comment:
It looks like the [desktop app through the app store](https://cloudup.com/c_LGPG1CfXB) still uses this method of determining chat eligibility. We should probably either update that environment's config or update this to use the purchases endpoint if we still need Olark for that environment.